### PR TITLE
feat: add `bones_camera_shake` crate

### PR DESCRIPTION
**Pull request by Piturnah**
As discussed on Discord, here is the camera shake functionality from [Bomby](https://github.com/fishfolk/bomby). For the time being it uses `bevy_ecs` and not `bones_ecs`.

### CODE REVIEW

File: crates/bones_camera_shake/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+repository = "https://github.com/fishfolk/bones"
+
+[dependencies]
+bevy = { version = "0.9", default_features = false }
+# For now we are pointing at a git rev but this should be rectified when a new release is made.
+# See https://github.com/Razaekel/noise-rs/pull/281.
+noise = { git = "https://github.com/bsurmanski/noise-rs", rev = "5abdde1b819eccc47e74969c15e1b56ae5a055d6" }
+
+[dev-dependencies]
+bevy_core_pipeline = "0.9"
```
Comment (zicklag) [inline crates/bones_camera_shake/Cargo.toml:18]: Is this needed here?

**Review by zicklag — COMMENTED** [review]
Just one question, otherwise this looks good, thanks!

File: crates/bones_camera_shake/Cargo.toml (hunk 1 above)
Comment (Piturnah) [reply crates/bones_camera_shake/Cargo.toml:18]: This dependency is required for the [top level example](https://github.com/Piturnah/bones/blob/3f87256f22896ece56f9e06a80adb1a3362e9c45/crates/bones_camera_shake/src/lib.rs#L6) or it won't compile as certain bevy features that aren't required for the core functionality are missing. Alternatively we could tell rustc to ignore that example or remove the example.

Comment (zicklag) [reply crates/bones_camera_shake/Cargo.toml:18]: Ah, I figured it was useful for examples, but I didn't see any. I had forgotten about doc examples.

In that case, it makes sense and I think this is good to go.

**Comment by zicklag** [comment]
bors merge

**Comment by zicklag** [comment]
bors help

**Comment by Piturnah** [comment]
Maybe you have to approve the PR?

**Review by zicklag — APPROVED** [review]

**Comment by zicklag** [comment]
bors merge

**Comment by zicklag** [comment]
Very strange. I'll try changing the branch protection rules.

bors merge

**Comment by zicklag** [comment]
Looks like it might be a fluke or a bug. I'll try one more time before just merging it manually.

https://forum.bors.tech/t/rejected-by-pr-status-despite-the-check-is-validated/645

bors merge

**Pull request merged**